vignettes/pkgnet-gallery.Rmd

```diff
@@ -0,0 +1,20 @@
+---
+title: "Gallery"
+output: 
+    rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{}
+  %\VignetteEncoding{UTF-8}
+---
+Welcome to the **pkgnet** gallery!   Click one any item below to see an example report.  Go to the [github page](https://github.com/UptakeOpenSource/pkgnet) to contribute. 
+
+<!--html_preserve-->
+<table>
+    <tr valign="top">
+        <td width="33%">lubridate<br><a href="https://raw.githubusercontent.com/UptakeOpenSource/pkgnet/master/gallery/pkgnet_gallery_lubridate.html"><img width="300" src="https://raw.githubusercontent.com/UptakeOpenSource/pkgnet/master/gallery/pkgnet_gallery_lubridate.png"></a></td>
+        <td width="33%">pkgnet<br><a href="https://raw.githubusercontent.com/UptakeOpenSource/pkgnet/master/gallery/pkgnet_gallery_pkgnet.html"><img width="300" src="https://raw.githubusercontent.com/UptakeOpenSource/pkgnet/master/gallery/pkgnet_gallery_pkgnet.png"></a></td>
+        <td width="33%">updraft<br><a href="https://raw.githubusercontent.com/UptakeOpenSource/pkgnet/master/gallery/pkgnet_gallery_updraft.html"><img width="300" src="https://raw.githubusercontent.com/UptakeOpenSource/pkgnet/master/gallery/pkgnet_gallery_updraft.png"></a></td>
+    </tr>
+</table>
+<!--/html_preserve-->
```

Comment:
@jayqi , long term, I would prefer that those who wish to contribute to the gallery host their own `pkgnet` reports and simply provide the image and link in this gallery.  This will share the love...and the burden of hosting the files. 

By code-gening this section, I believe that would make that interaction more difficult.  I also think that adding 1-2 lines of code here is not too much to ask of a contributor,

Reply:
Also, @jayqi @jameslamb , I have not been trying to match the website with the CRAN version.  I think we should keep it up to date with the dev version.  This seems to be the convention with half the open source R package websites I find.

Reply:
That convention makes no sense to me. The website should be an expected way for people to consume documentation for a package. If that is inconsistent with the preferred, most-likely version that a user has, then that is terrible UX.

Reply:
@bburns632 I'd be happy to review whenever you get back to the changes the three of us talked about tonight IRL. Sorry I didn't chip in on the first round of reviews.